packages/angular_devkit/build_angular/src/webpack/plugins/karma/karma.ts

```diff
@@ -336,6 +339,13 @@ function fallbackMiddleware() {
 function isPlugin(moduleId: string, pluginName: string) {
```

Comment:
Since we know the module name/plugin that we are looking for. We can optimize and simplify this a bit.

```ts
function isPlugin(moduleId: string, pluginName: string) {
  return (plugin: string|{}): boolean => {
    if (typeof plugin === 'string') {
      if (!plugin.includes('*')) {
        return plugin === moduleId;
      }

      const regexp = new RegExp(`^${plugin.replace('*', '.*')}`);
      if (regexp.test(moduleId)) {
        try {
          require.resolve(moduleId);

          return true;
        } catch {}
      }

      return false;
    }

    return pluginName in plugin;
  }
}

```

Reply:
Thanks for the suggestion. Fixed in 7434f5d

Reply:
Thanks for the suggestion. Fixed in 7434f5d